### PR TITLE
Update EIP-1898: Move to draft

### PR DESCRIPTION
**Pull request by macfarla**
Signed-off-by: Sally MacFarlane <macfarla.github@gmail.com>

It is supported by Geth 1.9.6 ([PR](https://github.com/ethereum/go-ethereum/pull/19491)).
It is supported by Besu 20.10.4 ([PR](https://github.com/hyperledger/besu/pull/1784)).

When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Review by charles-cooper — DISMISSED** [review]

**Comment by charles-cooper** [comment]
This could probably be moved to review or final stage. It has been implemented and stable in clients for years now.

**Comment by macfarla** [comment]
> Details

Thanks @charles-cooper - changed to Review

File: EIPS/eip-1898.md [hunk 1]
```diff
@@ -1,39 +1,42 @@
 ---
 eip: 1898
-title: Add `blockHash` to JSON-RPC methods which accept a default block parameter.
+title: Add `blockHash` to defaultBlock methods
+description: Add `blockHash` option to JSON-RPC methods that currently support defaultBlock parameter.
 author: Charles Cooper (@charles-cooper)
+discussions-to: https://ethereum-magicians.org/t/eip-1898-add-blockhash-option-to-json-rpc-methods-that-currently-support-defaultblock-parameter/11757
+status: Review
```
Comment (Pandapip1) [inline EIPS/eip-1898.md:7]:
```suggestion
status: Draft
```
Stagnant EIPs cannot be moved to Review. They must go through the entire EIP process starting from Draft.

File: EIPS/eip-1898.md [hunk 2]
```diff
@@ -1,39 +1,42 @@
...
 type: Standards Track
 category: Interface
-status: Stagnant
 created: 2019-04-01
 requires: 234, 1474
```
Comment (Pandapip1) [inline EIPS/eip-1898.md:11]: EIP-1474 needs to be pushed forward, too.

File: EIPS/eip-1898.md (hunk 1 above)
Comment (macfarla) [reply EIPS/eip-1898.md]: Got it. thanks!

File: EIPS/eip-1898.md (hunk 2 above)
Comment (macfarla) [reply EIPS/eip-1898.md:11]: opened a separate PR for this https://github.com/ethereum/EIPs/pull/5986

**Review by charles-cooper — APPROVED** [review]

**Comment by macfarla** [comment]
opened a separate PR for moving EIP-1474 back to draft also https://github.com/ethereum/EIPs/pull/5986

**Comment by xinbenlv** [comment]
LGTM: the author approved and the update seems reasonable.

**Review by Pandapip1 — APPROVED** [review]

**Pull request merged**